Add ability to specify default settings values from the registry on Windows (#404)
microsoft/Git-Credential-Manager-Core: docs/environment.md

[Git Credential Manager Core](usage.md) works out of the box for most users. Configuration options are available to customize or tweak behavior.

> Git Credential Manager Core (GCM Core) can be configured using environment variables. **Environment variables take precedence over [configuration](configuration.md) options**. System administrators may also configure [default values](enterprise-config.md) for many settings used by GCM Core.

Reviewer: Perhaps separate this from the earlier discussion by just saying that environment variables have highest priority?

```suggestion
Git Credential Manager Core (GCM Core) can be configured using environment variables. **Environment variables take precedence over any other configuration options.**
```
